components/HamburgerMenu.vue

```diff
@@ -0,0 +1,109 @@
+                            <path stroke-width="3" fill="none" d="M6.25,6.25,17.75,17.75" />
+                            <path stroke-width="3" fill="none" d="M6.25,17.75,17.75,6.25" />
+                        </svg>
+                    </button>
+                </div>
+                <div id="language-section" class="flex mt-2 px-5">
+                    <span class="text-primary-text self-center mr-2">{{ $t('hamburgerMenu.languageDropdownTitle') }}</span>
+                    <LocaleSelector class="py-1.5 text-primary-text bg-primary-bg" />
+                </div>
+                <div id="menu-items" class="mt-10 px-5 flex flex-col gap-6">
+                    <NuxtLink :to="'/'">
+                        <div @click="toggleMenu()">
+                            {{ $t('hamburgerMenu.home') }}
+                        </div>
+                    </NuxtLink>
+                    <NuxtLink :to="'/about'">
+                        <div @click="toggleMenu()">
+                            {{ $t('hamburgerMenu.about') }}
+                        </div>
+                    </NuxtLink>
+                    <NuxtLink :to="'https://forms.gle/4E763qfaq46kEsn99'" target="_blank">
+                        <div @click="toggleMenu()">
+                            {{ $t('hamburgerMenu.contact') }}
+                        </div>
+                    </NuxtLink>
+                    <NuxtLink :to="'/submit'">
+                        <div @click="toggleMenu()">
+                            {{ $t('hamburgerMenu.submit') }}
+                        </div>
+                    </NuxtLink>
+                </div>
+            </div>
+
+            <div id="footer-section" class="flex flex-col gap-5 px-5">
+                <div id="row-1" class="flex gap-4">
+                    <NuxtLink :to="'/terms'">
+                        <span @click="toggleMenu()">
+                            {{ $t('footer.terms') }}
+                        </span>
+                    </NuxtLink>
+                    <NuxtLink :to="'/privacypolicy'">
+                        <span @click="toggleMenu()">
+                            {{ $t('footer.privacy') }}
+                        </span>
+                    </NuxtLink>
+                </div>
+                <div id="row-2" class="flex gap-6">
+                    <NuxtLink :to="'https://github.com/ourjapanlife'" target="_blank">
+                        <SVGGithubIcon id="github-icon" alt="hamburger menu" class="h-10 w-8" />
+                    </NuxtLink>
+                    <!-- Netlify Icons are available here: https://www.netlify.com/press/#badges -->
+                    <NuxtLink :to="'https://www.netlify.com'" target="_blank">
+                        <img src="https://www.netlify.com/v3/img/components/netlify-light.svg" alt="Deploys by Netlify"
+                            class=" h-10 w-20" />
+                    </NuxtLink>
```

Comment:
I think it would be good to include a test that the Netlify element is always present in the hamburger menu.

Reply:
makes sense to me. I'll add it later tonight